### PR TITLE
Add section for International projects

### PR DESCRIPTION
**Pull request by nschonni**
May have some overlap with https://github.com/opengovplatform that is currently under the US section

### CODE REVIEW

**Comment by afeld** [comment]
Interesting... from a quick glance, it seems that http://ogpl.gov.in/ and http://opengovplatform.org/ are the same site under different domains?

**Comment by afeld** [comment]
/cc a couple random people from their contributor pages who might be able to give more insight on the distinction: @pjackson28 @subhransusekhar

I don't think having duplicates in the list is helpful, though it seems like the repositories are separate.  Probably good feedback that the relationship between the two organizations is confusing, though :wink:

**Comment by pjackson28** [comment]
https://github.com/opengovplatform and https://github.com/ogpl are associated but different. Essentially https://github.com/ogpl is seeking to bridge the various projects associated to OGPL (Open Government Platform) whereas https://github.com/opengovplatform is one of the associated projects.

As for http://ogpl.gov.in and http://opengovplatform.org, I believe they are the same site with different domains. Much like with https://github.com/opengovplatform, http://opengovplatform.org/ and http://ogpl.gov.in are associated to https://github.com/ogpl but are different as well (they predate https://github.com/ogpl which will have http://ogpl.github.io for its site).

**Comment by afeld** [comment]
@pjackson28 Thanks for the explanation! I'm still a bit confused about the relationships of the various sites/orgs, but that's beside the point :wink:

File: _config.yml [hunk 1]
```diff
@@ -87,6 +87,9 @@ organizations:
   Estonia:
     - vvk-ehk
 
+  International
```
Comment (benbalter) [inline _config.yml:90]: Just a heads up, valid YAML dictates a trailing `:` here. Fixed in 3dee3d1.

Comment (nschonni) [inline _config.yml:90]: Thanks, that's what I get for not going to bed at a human hour :frowning:

**Pull request merged**